packages/carousel/src/ArrowButton/ArrowButton.styles.js

```diff
@@ -46,7 +46,7 @@ const topBottomStyles = (props) =>
       opacity: 40%;
       color: ${getPaletteColor('border.base')(props)} !important;
       background: ${getPaletteColor('background.lightest')(props)} !important;
-      box-shadow: ${props.theme.boxShadows[0]};
+      box-shadow: ${props.theme.shadows[0]};
```

Comment:
should this be a named shadow @wesleymartin85 ?

Reply:
@craigpalermo considering the carousel came up in the change sets, did you already update this to use one of our named shadows?

Reply:
@craigpalermo md